filebeat/module/system/syslog/ingest/pipeline.json

```diff
@@ -36,10 +36,18 @@
                     "MMM dd HH:mm:ss",
                     "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZZ"
                 ],
-                {< if .convert_timezone >}"timezone": "{{ event.timezone }}",{< end >}
                 "ignore_failure": true
             }
         },
+        {
+            "date": {
+                "if": "ctx.event.timezone != null",
+                "field": "@timestamp",
+                "formats": ["ISO8601"],
+                "timezone": "{{ event.timezone }}",
+                "on_failure": [{"set": {"field": "error", "value": "{{ _ingest.on_failure_message }}"}}]
```

Comment:
This should be `error.message` to be compliant with ECS?

Reply:
Yes.